protocol/app/msgs/normal_msgs.go

```diff
@@ -211,6 +212,65 @@ var (
 
 		// ica
 		"/ibc.applications.interchain_accounts.v1.InterchainAccount": nil,
+
+		// wasm
+		"/cosmwasm.wasm.v1.AcceptedMessageKeysFilter":                  nil,
```

Comment:
As commented [here](https://www.notion.so/dydx/CosmWasm-Implementation-Spec-aa96d7ae25c64ccaadc1f61827c69e73?pvs=4#0fb0e1cbe36e4506ba25342385a9dffc), we should audit these messages, only support necessary ones and move others to unsupported [messages](https://github.com/dydxprotocol/v4-chain/blob/b0ca05073a250992478a39c3614f081cbdf79d46/protocol/lib/ante/unsupported_msgs.go#L11). Could you add a TODO item for this?

Reply:
There is a todo item in linear https://linear.app/dydx/issue/OTE-461/audit-messages-allowed-by-cosmwasm

Reply:
I'll add a comment for TODO

Reply:
Note clarified the linear ticket a bit more to disambiguate from another audit.